src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -124,11 +124,20 @@ public static bool CanPutCommand(string command)
 
         static ClusterCommandsVersionManager()
         {
-            MyCommandsVersion = _currentClusterMinimalVersion = Enumerable.Max(ClusterCommandsVersions.Values);
```

Comment:
Who will set`_currentClusterMinimalVersion` now?

Reply:
This is been set from `SetClusterVersion` and it is called either by the Leader or by a Follower after negotiated with the leader about the cluster version.

Reply:
okay